server/src/main/java/org/opensearch/rest/action/search/RestMultiSearchAction.java

```diff
@@ -158,6 +159,7 @@ public static MultiSearchRequest parseRequest(RestRequest restRequest,
             multiRequest.add(searchRequest);
         });
         List<SearchRequest> requests = multiRequest.requests();
+        final TimeValue cancelAfterTime = restRequest.paramAsTime("cancel_after_timeinterval", null);
```

Comment:
For naming consistency:
```suggestion
        final TimeValue cancelAfterTimeInterval = restRequest.paramAsTime("cancel_after_time_interval", null);
```